### PR TITLE
Added documentation for the 3ddistance flag

### PR DESCRIPTION
**Pull request by MaxRobinsonTheGreat**
May want to wait until test is made to merge

### CODE REVIEW

File: docs/packages/docs/tasks/distance-task.rst [hunk 1]
```diff
@@ -60,6 +60,15 @@ If left empty, it defaults to false.
...
 ``"MaximizeDistance": true``
 
+3dDistance
+~~~~~~~~~~
+
+Boolean value to indicate whether to incorporate height into the distance calculation.
+If false, it will only use the xy values and ignore vertical distance.
+If left empty, it defaults to false.
+
+``"3dDistance": true``
```
Comment (jaydenmilne) [inline docs/packages/docs/tasks/distance-task.rst]: Ugh our capitalization scheme has failed us

File: docs/changelog/changelog.rst [hunk 2]
```diff
@@ -21,7 +21,10 @@ Highlights
 
 New Features
 ~~~~~~~~~~~~
-
+- :ref:`distance-task` can now be set to use only the xy coordinates in the
```
Comment (jaydenmilne) [inline docs/changelog/changelog.rst]: Suggested rewrite (to make default behavior more clear)
```
- :ref:`distance-task` by default now calculates the distance to the objective along the XY plane, to discourage flying straight up.

  If the full 3D distance is desired, set the ``3dDistance`` flag in the configuration block of the :ref:`distance-task`.
(`#360 <https://github.com/BYU-PCCL/holodeck/issues/360>`_)
```

Comment (MaxRobinsonTheGreat) [reply docs/changelog/changelog.rst]: As per usual I like your rewrite more than mine 😞

File: docs/packages/docs/tasks/distance-task.rst (hunk 1 above)
Comment (MaxRobinsonTheGreat) [reply docs/packages/docs/tasks/distance-task.rst]: get over it bro

**Pull request merged**